app/models/user.rb

```diff
@@ -1,3 +1,14 @@
 class User < ActiveRecord::Base
   has_many :cards, dependent: :destroy
+  has_many :authentications, dependent: :destroy
+  accepts_nested_attributes_for :authentications
+
+  validates :password,                presence: true, confirmation: true, length: { minimum: 3}
```

Comment:
Line is too long. [95/80]<br>Unnecessary spacing detected.<br>Space inside } missing.